client/galaxy/scripts/mvc/workflow/workflow-forms.js

```diff
@@ -194,8 +194,10 @@ function _visit(head, head_list, output_id, options) {
 /** Builds sub section with step actions/annotation */
 function _makeSection(output_id, options) {
     var extensions = [];
+    var dynamicActionOptions = [];
     var input_terminal_names = [];
     var datatypes = options.datatypes;
+    var dynamicActions = options.dynamicActions;
```

Comment:
This is correct convention but in this file we are using underscores for variable names i.e. "dynamic_actions". Can you adjust this for consistency?